src/styles.scss

```diff
@@ -1,24 +1,25 @@
-$primary: #0059F7;
```

Comment:
This was all from Prettier 😬 I think it helps with consistency and readability to have these changes

Apart from all the syntax updates, I removed the following styles and added them to `log-in.component.scss` since that's the only place they were being used 
```scss
.log-in-google,
.log-in-seed,
.sign-up-google,
.sign-up-seed {
  height: 72px;
  width: 290px;
  background-size: 290px 72px;
}

.log-in-google {
  background-image: url(/assets/log_in_google.png);
}

.log-in-seed {
  background-image: url(/assets/log_in_seed.png);
}

.sign-up-google {
  background-image: url(/assets/sign_up_google.png);
}

.sign-up-seed {
  background-image: url(/assets/sign_up_seed.png);
}
```